netbox/core/management/commands/rqworker.py

```diff
@@ -14,6 +17,16 @@ class Command(_Command):
     of only the 'default' queue).
     """
     def handle(self, *args, **options):
+        # Setup system jobs.
+        for job in registry['system_jobs'].values():
+            if getattr(job.Meta, 'system_enabled', True):
```

Comment:
Could we do away with the `system_enabled` attribute and just check whether `system_interval` is defined? The former seems redundant since we're already registering system jobs automatically.